bin/varnishd/cache/cache_rfc2616.c

```diff
@@ -220,6 +220,17 @@ RFC2616_Ttl(struct busyobj *bo, vtim_real now, vtim_real *t_origin,
 		*grace = rfc2616_time(p);
 	}
 
+	/*
+	 * The "must-revalidate" directive prohibits 
+	 * the cache from serving stale data,
+	 * according to RFC7234, 5.2.2.1.
+	 * This sets the grace to zero.
+	 */
```

Comment:
Not sure we need this comment now that we have an RFC reference, and the code is rather self-explanatory.

Reply:
@Dridi I removed that part.